flink-runtime/src/main/java/org/apache/flink/runtime/jobmaster/slotpool/SlotPoolGateway.java

```diff
@@ -108,7 +108,7 @@
 	 * individually accepted or rejected by returning the collection of accepted
 	 * slot offers.
 	 *
-	 * @param taskManagerLocation from which the slot offers originate
+	 * @param taskManagerLocation from which the slot offer originates
```

Comment:
this method on the other hand offers multiple slots, so plural makes more sense.